widgets/GoogleDrive-batchsink.json

```diff
@@ -0,0 +1,54 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "Google Drive sink",
+  "configuration-groups": [
+    {
+      "label": "Basic",
```

Comment:
Need to have two sections:
Basic:
- Reference Name
- Schema field used as name of file
- Schema field used as body of file

Reply:
done